GameDataParser/Parsers/ItemParser.cs

```diff
@@ -3,42 +3,56 @@
 using System.Diagnostics;
 using System.IO;
 using System.IO.MemoryMappedFiles;
-using System.Linq;
 using System.Xml;
 using GameDataParser.Crypto.Common;
+using Maple2Storage.Types.Metadata;
 using Maple2Storage.Types;
 using ProtoBuf;
+using System.Configuration;
 
-namespace GameDataParser.Parsers {
-    public static class ItemParser {
-        private const string OUTPUT = "ms2-item-metadata";
+namespace GameDataParser.Parsers
+{
+    public static class ItemParser
+    {
+        private static readonly string OUTPUT = ConfigurationManager.AppSettings["OUTPUT"] + "/ms2-item-metadata";
```

Comment:
Is there a reason we have a file for some of these settings but all, like in this case, `OUTPUT`?

Reply:
Becuase it will export the file into `/bin/debug/netappcore3.1/Resource` and instead, setting it to export on `GameDataParser/Resources`, setting at the config, will let people just to rename the last string to be the name of the file